### PR TITLE
OAS-4200: added locked resources info

### PR DESCRIPTION
**Pull request by ansoboleva**

### CODE REVIEW

File: oasis/access-control.md [hunk 1]
```diff
@@ -1,39 +1,48 @@
...
-For example, you can *create* a project in an organization. This requires the
-respective **permission**, which can be granted to organization members via a
-**role** such as the pre-defined *Project Administrator*.
+For example, you can create a project in an organization and create a deployment inside a project.
 
-The association of a member with a role is called a **role binding**.
-All role bindings of a resource comprise a **policy**.
+## Locked resources
+
+In ArangoDB Oasis, you can lock the resources and some of their properties to prevent accidental deletion. When an object is locked,
```
Comment (ewoutp) [inline oasis/access-control.md:20]: Please remove "and some of their properties"

"When an object is locked" -> "When a resource is locked"

File: oasis/access-control.md [hunk 2]
```diff
@@ -1,39 +1,48 @@
...
+it cannot be deleted and must be unlocked first.
 
-Roles can be bound on an organization, project and deployment level (listed in the high to low level order). The lower
-levels *inherit* permissions from their parents.
+The hierarchial structure of the resources (organization-project-deployment) is used in the locking functionality: if a child resource is locked
+(for example, a project), you cannot delete the parent organization without unlocking that project first.
```
Comment (ewoutp) [inline oasis/access-control.md:24]: Prefer to use the relation between project & deployment in this example.

... "you cannot delete the parent project without unlocking that deployment first"

File: oasis/access-control.md [hunk 3]
```diff
@@ -1,39 +1,48 @@
...
 
-A policy is a set of bindings of roles to users or groups for a specific
-resource. This means that there is a unique policy per resource (an organization, a project or a deployment).
+Roles can be bound on an organization, project and deployment level (listed in the high to low level order, with lower
+levels inheriting permissions from their parents). This means that there is a unique policy per resource (an organization, a project or a deployment).
 
 For example, an organization has exactly one policy,
 which binds roles to members of the organization. These bindings are used to
-give the users permissions to perform operations on this organization.
+give the users permissions to perform operations in this organization.
```
Comment (ewoutp) [inline oasis/access-control.md:45]: Not sure this change is correct.

Technically the permissions give access to operations ON the organization, but also to operations on resources in the organization.

**Review by ewoutp — CHANGES_REQUESTED** [review]

File: oasis/access-control.md (hunk 3 above)
Comment (ansoboleva) [reply oasis/access-control.md:45]: "to operations on resources in the organization" - yes, exactly. But if you have a hard opinion and want to use "on", I can change it back.

**Review by ewoutp — APPROVED** [review]
LGTM

**Pull request merged**